processor/processor.go

```diff
@@ -509,7 +508,7 @@ func loadConfig() {
 	config.RegisterBoolConfigVariable(false, &enableEventSchemasAPIOnly, true, "EventSchemas.enableEventSchemasAPIOnly")
 	config.RegisterIntConfigVariable(10000, &maxEventsToProcess, true, 1, "Processor.maxLoopProcessEvents")
 
-	batchDestinations, customDestinations = misc.LoadDestinations()
+	batchDestinations, _ = misc.LoadDestinations()
```

Comment:
since `misc.LoadDestinations` is only used here and we don't really care about custom destinations, please simplify it by renaming it to `misc.BatchDestinations` and  only return a single value
```suggestion
	batchDestinations = misc.BatchDestinations()
```

Reply:
Addressed